README.md

```diff
@@ -4,6 +4,10 @@ For detailed information on this package, please refer to the [online documentat
 
 ## Release History
 
+### Version next
+
+* Corrected the ElastiCache cache hit rate metric FQN for element details dashboard
```

Comment:
@tombakerjr could you indicate that you corrected this in the policy, too?

Reply:
Done